components/ipfs/ipfs_switches.h

```diff
@@ -1,17 +0,0 @@
-/* Copyright 2020 The Brave Authors. All rights reserved.
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#ifndef BRAVE_COMPONENTS_IPFS_IPFS_SWITCHES_H_
-#define BRAVE_COMPONENTS_IPFS_IPFS_SWITCHES_H_
-
-namespace ipfs {
-
-// Allows disabling the Ipfs client updater extension.
-constexpr char kDisableIpfsClientUpdaterExtension[] =
```

Comment:
This command-line switch is removed because we already have a feature flag to control IPFS.